Makefile

```diff
@@ -32,14 +33,30 @@ $(call add-bindata,v3.11.0,./bindata/v3.11.0/...,bindata,v311_00_assets,pkg/oper
 clean:
 	$(RM) ./cluster-svcat-apiserver-operator
 .PHONY: clean
-
-
-update-codegen-crds:
-	go run ./vendor/github.com/openshift/library-go/cmd/crd-schema-gen/main.go --domain openshift.io --apis-dir vendor/github.com/openshift/api
+GO_TEST_PACKAGES :=./pkg/... ./cmd/...
+
+.PHONY: test-e2e
+test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
+test-e2e: test-unit
+
+CRD_SCHEMA_GEN_VERSION := v1.0.0
+crd-schema-gen:
+	git clone -b $(CRD_SCHEMA_GEN_VERSION) --single-branch --depth 1 https://github.com/openshift/crd-schema-gen.git $(CRD_SCHEMA_GEN_GOPATH)/src/github.com/openshift/crd-schema-gen
```

Comment:
Is it possible to vendor the source for `github.com/openshift/crd-schema-gen` and then just `go run` it like we did before from `library-go`?

Reply:
While I agree it's very odd, it follows the patterns of some other operators already.
https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/Makefile#L41-L44

https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/Makefile#L44-L48

https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/Makefile#L45-L48

Reply:
Ok no worries. Just thought I'd ask.

Also, if this were a typical operator that we planned to maintain for awhile, I'd also ask if we could make the tmpdir deterministic and local to the project so that subsequent runs could reuse the built binary if already present. But since this has a short shelf life, I'll leave that up to you.

Reply:
Yeah, I'd rather not make that change right now.